samples/framework/libraries/migrate-library-csproj-keep-existing/src/Car/Car.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <AssemblyName>Car</AssemblyName>
+    <PackageId>Car</PackageId>
+    <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
+    <PackageTargetFallback>$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
Latest version is 10.0.1 ... https://www.nuget.org/packages/Newtonsoft.Json/

Looks like *packages.config* (above) will need this update.